routes/route.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+
+router.post("/",(req,res,next)=>{
+    req.file = req.body;
+    next();
+},(req,res,next)=>{
+    var file = req.file;
+    var stringfile = JSON.stringify(file.payload)
```

Comment:
express.json() middlware converts string to json. here in this line json to string. any ideas to avoid this to and fro conversion? for large sized files this will have performance hit.

Reply:
Hi Arul,

Now i am not converting data to string after express.json() i am iterating through json object and replacing the value.

Regards,
Amit